src/common/options.cc

```diff
@@ -6938,6 +6938,12 @@ std::vector<Option> get_mds_options() {
     Option("mds_hack_allow_loading_invalid_metadata", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
      .set_default(0)
      .set_description("INTENTIONALLY CAUSE DATA LOSS by bypasing checks for invalid metadata on disk. Allows testing repair tools."),
+    Option("mds_drop_cache_ack_timeout", Option::TYPE_UINT,
+	   Option::LEVEL_ADVANCED)
+     .set_default(30)
+     .set_description("how long should the MDS wait for receiving "
+		      "CEPH_SESSION_RECALL_STATEMSG_ACK from all sessions "
+		      "while dropping cache."),
```

Comment:
I think we can keep this vague. Don't mention dropping cache. Also, rename to `mds_session_recall_ack_timeout`.